static/app/views/issueDetails/groupDistributions/useDrawerTab.tsx

```diff
@@ -0,0 +1,16 @@
+import useUrlParams from 'sentry/utils/useUrlParams';
+import {DrawerTab} from 'sentry/views/issueDetails/groupDistributions/types';
+
+export default function useDrawerTab({enabled}: {enabled: boolean}) {
+  const {getParamValue: getTabParam, setParamValue: setTabParam} = useUrlParams(
+    'tab',
+    DrawerTab.TAGS
+  );
+
+  return enabled
+    ? {
+        tab: getTabParam() as DrawerTab,
```

Comment:
This `tab: getTabParam() as DrawerTab,` is a type cast, but maybe it should be stricter value check in case the user manually types something strange into the url.

I'll followup on that afterwards though.